### PR TITLE
go 1.15 

### PR DESCRIPTION
**Pull request by ajayk**
### Description

bump to go 1.15

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
- [x] Make sure the title of the PR is a good description that can go into the release notes

### CODE REVIEW

File: build/docker/build_image_manifest [hunk 1]
```diff
@@ -1,9 +1,6 @@
-"github.com/cloudflare/cfssl v0.0.0-20190726000631-633726f6bcb7"
```
Comment (michaelbeaumont) [inline build/docker/build_image_manifest:1]: How did this end up getting removed?

**Comment by michaelbeaumont** [comment]
@ajayk Do you think you'll have time to continue this PR?

**Comment by ajayk** [comment]
@michaelbeaumont yep probably will try to get this by the weekend

**Comment by neha-viswanathan** [comment]
@ajayk @michaelbeaumont is this still a WIP? can I contribute?

**Comment by ajayk** [comment]
> @ajayk @michaelbeaumont is this still a WIP? can I contribute?

@michaelbeaumont i am closing this one and i have opened up #2652 , 
@neha-viswanathan you can pick this up from #2652

**Pull request closed without merging**